man/sd-id128.xml

```diff
@@ -85,7 +87,7 @@
 
     <programlisting>#define SD_MESSAGE_COREDUMP SD_ID128_MAKE(fc,2e,22,bc,6e,e6,47,b6,b9,07,29,ab,34,a2,50,b1)</programlisting>
 
-    <para><constant>SD_ID128_NULL</constant> may be used to refer to the 128bit ID consisting of only
+    <para><constant>SD_ID128_NULL</constant> may be used to refer to the 128-bit ID consisting of only
```

Comment:
hmm, is that an accepted spelling? with the dash?

Reply:
Both spelling were used in this file. I think '128-bit' is better than '128bit'. I'd also even prefer '128 bit', but I made the minimal change.